pydata_sphinx_theme/docs-navbar.html

```diff
@@ -12,7 +12,7 @@
         <span class="navbar-toggler-icon"></span>
     </button>
 
-    <div id="navbar-menu" class="col-lg-9 collapse navbar-collapse">
+    <div id="navbar-menu" class="col-11 collapse navbar-collapse">
```

Comment:
here we make the topbar items snap to the left, instead of being centered in the middle of the topbar